docs/reference/generated/accordion-root.json

```diff
@@ -13,9 +13,9 @@
       "detailedType": "any[] | undefined"
     },
     "onValueChange": {
-      "type": "((value: any[]) => void)",
+      "type": "((value: any[], event: Accordion.Root.ChangeEventData) => void)",
```

Comment:
The parameter should be called `data`.
However, looking at the whole PR, I'm not sold on the `data` name. In particular, code like `data.cancel` looks ugly. Am I canceling data?
How about `eventData` or `eventDetails` instead?

Reply:
> However, looking at the whole PR, I'm not sold on the data name. In particular, code like data.cancel looks ugly. Am I canceling data?

Yeah, agreed; I preferred `event.cancel()` but we shouldn't mutate event objects.  `eventDetails.cancel()` seems to work.